src/mdns/mdns_avahi.cpp

```diff
@@ -952,8 +953,8 @@ void PublisherAvahi::ServiceSubscription::HandleBrowseResult(AvahiServiceBrowser
 
     assert(mServiceBrowser == aServiceBrowser);
 
-    otbrLogInfo("browse service reply: %s.%s proto %d inf %u event %d flags %u", aName, aType, aProtocol,
-                aInterfaceIndex, aEvent, aFlags);
+    otbrLogInfo("browse service reply: %s.%s proto %d inf %u event %d flags %d", aName, aType, aProtocol,
+                aInterfaceIndex, static_cast<int>(aEvent), static_cast<int>(aFlags));
```

Comment:
Not necessary to use `static_cast<int>` ?

Anything smaller than int will be converted to int automatically.

Reply:
Is there a reference about this behavior?

Reply:
fine to me with casting to int.